app/views/components/docs/page_preview.yml

```diff
@@ -8,7 +8,7 @@ accessibility_criteria: |
 examples:
   default:
     data:
-      title: "A carrot cake has been baked"
-      base_path: "/news/cake-baked"
-      description: A carrot cake was baked for the publishing workflow team's fika, the government announced today.
-      url: "https://www.gov.uk/browse"
+      title: Foreign Secretary heads to Brussels to discuss Iran
+      base_path: "/government/news/foreign-secretary-heads-to-brussels-to-discuss-iran"
+      description: Ahead of the Foreign Affairs Council in Brussels, Jeremy Hunt said he would be building on the leadership shown by the E3 to maintain the nuclear deal.
+      url: "https://draft-origin.publishing.service.gov.uk/government/news/foreign-secretary-heads-to-brussels-to-discuss-iran"
```

Comment:
This seems a lot less fun 😢

Reply:
I wanted to be able to do manual testing using an item from the draft stack. Happy to keep the old title and description though.

Reply:
No no, it's better to have something real. We'll just refer to you as the fun police 😉